runtimes/eden/src/lib.rs

```diff
@@ -93,13 +94,13 @@ construct_runtime! {
 		TechnicalMembership: pallet_membership::<Instance3> = 17,
 
 		// Consensus
+		CollatorSelection: pallet_collator_selection::{Pallet, Call, Storage, Event<T>, Config<T>} = 19,
```

Comment:
Can we use `26`? I was trying to keep the `1*` prefix for governance and `2*` for consensus

Reply:
In hindsight we could also reuse `21` or `22`

Reply:
I wanted to be very cautious to not reuse pallet ids on subsequent upgrades. This may trigger very weird bugs/issues in the tools which explore the blocks but don't update the metadata in the right place. Though it should be the concern of those tools, but I wanted not to leave any room for such baffling/misleading issues.

Reply:
Makes sense, shall we do `26` then?

Reply:
It's necessary for CollatorSelection to be declared before Session. I can still keep it before Session and give it the number 26 but that wouldn't look good and may tempt people to reorder it which will be breaking.